shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -567,6 +567,9 @@ - (void)setTextInputClient:(int)client withConfiguration:(NSDictionary*)configur
   _view.keyboardType = ToUIKeyboardType(configuration[@"inputType"]);
   _view.autocapitalizationType = ToUITextAutocapitalizationType(configuration[@"inputType"]);
   _view.secureTextEntry = [configuration[@"obscureText"] boolValue];
+  _view.autocorrectionType = [configuration[@"autocorrect"] boolValue]
```

Comment:
Yeah, was wondering about that, but I'm assuming this will roll into the framework after flutter/flutter#11180 lands. Probably not a bad idea to add a paranoid check for it being missing though.